src/core/server/saved_objects/migrations/core/elastic_index.test.ts

```diff
@@ -415,11 +415,31 @@ describe('ElasticIndex', () => {
           query: {
             bool: {
               must_not: [
+                {
```

Comment:
This test probably needs to change so that we don't have to update every time a new type is added to `REMOVED_TYPES`